sdk/loadtestservice/azure-developer-loadtesting/azure/developer/loadtesting/aio/operations/_operations.py

```diff
@@ -2067,228 +2114,199 @@ async def get_load_test(self, test_id: str, **kwargs: Any) -> JSON:
+                            "unit": "str"  # Optional. Metric unit.
                         }
-                    ],
-                    "nextLink": "str"  # Optional. Link for the next list of resources in case of
-                      paginated results, if applicable.
+                    },
+                    "testId": "str"  # Optional. Test identifier.
+                }
+        """
+
+    @overload
+    async def create_or_update_server_metrics_config(
+        self, test_id: str, body: IO, *, content_type: str = "application/merge-patch+json", **kwargs: Any
+    ) -> JSON:
+        """Configure server metrics for a test.
+
+        Configure server metrics for a test.
+
+        :param test_id: Unique name for the load test, must contain only lower-case alphabetic,
+         numeric, underscore or hyphen characters. Required.
+        :type test_id: str
+        :param body: Server metric configuration model. Required.
+        :type body: IO
+        :keyword content_type: Body Parameter content-type. Content type parameter for binary body.
+         Default value is "application/merge-patch+json".
+        :paramtype content_type: str
+        :return: JSON object
+        :rtype: JSON
+        :raises ~azure.core.exceptions.HttpResponseError:
+
+        Example:
+            .. code-block:: python
+
+                # response body for status code(s): 200, 201
+                response == {
+                    "createdBy": "str",  # Optional. The user that created.
+                    "createdDateTime": "2020-02-20 00:00:00",  # Optional. The creation
+                      datetime(ISO 8601 literal format).
+                    "lastModifiedBy": "str",  # Optional. The user that last modified.
+                    "lastModifiedDateTime": "2020-02-20 00:00:00",  # Optional. The last Modified
+                      datetime(ISO 8601 literal format).
+                    "metrics": {
+                        "str": {
+                            "aggregation": "str",  # Metric aggregation. Required.
+                            "metricNamespace": "str",  # Metric name space. Required.
+                            "name": "str",  # The invariant value of metric name.
+                              Required.
+                            "resourceId": "str",  # Azure resource id. Required.
+                            "resourceType": "str",  # Azure resource type. Required.
+                            "displayDescription": "str",  # Optional. Metric description.
+                            "id": "str",  # Optional. Unique name for metric.
+                            "unit": "str"  # Optional. Metric unit.
+                        }
+                    },
+                    "testId": "str"  # Optional. Test identifier.
+                }
+        """
+
+    @distributed_trace_async
+    async def create_or_update_server_metrics_config(self, test_id: str, body: Union[JSON, IO], **kwargs: Any) -> JSON:
```

Comment:
i do also have a question about all of the patched operations. Why are so many being patched, are they not part of the swagger?